config/formats.js

```diff
@@ -568,6 +568,15 @@ exports.Formats = [
 		mod: 'oldweather',
 		ruleset: ['[Gen 7] OU'],
 	},
+	{
+		name: "[Gen 7] LU",
+		desc: ["The new tiering format that nobody wanted! This format is much more chaotic compared to OU and will be tiered differently. Ran by Lycanium Z. "],
```

Comment:
Shouldn't "Ran by Lycanium Z" say ~~"ran by the guy with too many names"~~ i mean "Ran by Worse Than Death"

Reply:
@WorseThanDeath pinging you so you see this message

Reply:
Yeah, I’m moving my 
 account to a new email HECKIN spam